### PR TITLE
[typo fix]: fix typo for some comment and method name

### PR DESCRIPTION
**Pull request by jxd134**
branch: main
target: fix typo

### CODE REVIEW

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 177 Deploy Test Success

**Review by benja-wu — APPROVED** [review]
LGTM

File: pkg/object/httpserver/httpserver.go [hunk 1]
```diff
@@ -81,7 +81,7 @@ func (hs *HTTPServer) Inherit(superSpec *supervisor.Spec, previousGeneration sup
 	}
 }
 
-// Status is the wrapper of runtime's Status.
+// Status is the wrapper of runtimes Status.
```
Comment (localvar) [inline pkg/object/httpserver/httpserver.go:84]: this one is not a typo.

**Comment by benja-wu** [comment]
@jxd134 

Auto testing failed due to the 
``` bash
# github.com/megaease/easegress/pkg/common [github.com/megaease/easegress/pkg/common.test]
272
pkg/common/signal_unix_test.go:30:42: undefined: SingalUsr2
273
# github.com/megaease/easegress/pkg/cluster [github.com/megaease/easegress/pkg/cluster.test]
274
FAIL	github.com/megaease/easegress/pkg/cluster [build failed]
275
pkg/cluster/cluster_test.go:358:11: undefined: isDataEuqal
276
FAIL	github.com/megaease/easegress/pkg/common [build failed]
277
pkg/cluster/cluster_test.go:374:10: undefined: isDataEuqal
```
You can check it locally by running `make test`.

File: pkg/object/httpserver/httpserver.go (hunk 1 above)
Comment (jxd134) [reply pkg/object/httpserver/httpserver.go:84]: yes, it's a mistake

**Comment by jxd134** [comment]
@benja-wu 
Yes, it's a mistake.
I will fix it.

**Review by benja-wu — CHANGES_REQUESTED** [review]
Test cases are also needed to be updated.

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 177 Deploy Test Success

**Comment by benja-wu** [comment]
Hi @jxd134 
I think this PR is very helpful to improve the code quality of Easegress.
Can you tell us the reason why u closed this PR? Thx.

**Comment by jxd134** [comment]
@benja-wu 
The code quality is poor.
I will fix it and open a new puul request.

**Comment by benja-wu** [comment]
> @benja-wu
> The code quality is poor.
> I will fix it and open a new puul request.

Roger that!

**Pull request closed without merging**